packages/ui/src/dropdown/single-select.tsx

```diff
@@ -110,7 +110,7 @@ export const Dropdown: FC<ISingleSelectDropdown> = (props) => {
   // hooks
   const handleKeyDown = useDropdownKeyPressed(toggleDropdown, handleClose);
 
-  useOutsideClickDetector(dropdownRef, handleClose);
+  useOutsideClickDetector(dropdownRef, handleClose, true);
```

Comment:
please test in all references of this drop down and check if the components around it are working fine as well. Cuz this could impact their functionality